packages/graphql/src/translate/translate-top-level-cypher.ts

```diff
@@ -172,8 +172,8 @@ export function translateTopLevelCypher({
     } else {
         cypherStrs.push(`
             CALL apoc.cypher.doIt("${statement}", ${apocParamsStr}) YIELD value
-            WITH apoc.map.values(value, [keys(value)[0]])[0] AS this
-        `);
+            WITH [k in keys(value) | value[k]][0] as this
```

Comment:
Not blocking but is the convention not for `as` to be upper case?

Reply:
Good catch
```suggestion
            WITH [k in keys(value) | value[k]][0] AS this
```